src/ActivityMammal/TrainWithExampleMammal/ChooseExample.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
```

Comment:
This file is a copy of the file `src/Activity/TrainWithExamples/ChooseExample`. The comment I made on duplicating three times the code of a function in your previous PR stands even more here. 

The activity for choosing an example works in the same way for  Parallelograms or Mammals so this should be a single component. The only difference is the property that makes the component display either the parallelogram examples or the mammal examples

Reply:
You should only add code if it brings new logic. Here you should modify the existing code to allow for handling several topics, not duplicate the code.